geomesa-kafka/geomesa-kafka-tools/conf-filtered/confluent-dependencies.sh

```diff
@@ -14,7 +14,7 @@
 confluent_install_version="%%confluent.recommended.version%%"
 
 function dependencies() {
-  local classpath="$1"
+  # local classpath="$1"
```

Comment:
I was just leaving it there as kind of a function definition. it would probably be better as a comment above the function, though - I'll update it. (the `dependencies` function gets injected per back-end, but classpath isn't used in all of them).